extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -88,17 +88,15 @@ amp-carousel .amp-carousel-button.amp-disabled {
   top: 0;
   width: 100% !important;
   scroll-snap-type: mandatory !important;
-  -ms-overflow-style: none;
-  -webkit-overflow-scrolling: touch !important;
-}
-
-.-amp-slides-container::-webkit-scrollbar {
   /**
-   * Height not set to 0 because of safari bug.
-   * https://bugs.webkit.org/show_bug.cgi?id=160622
+   * Hide the scrollbar by tucking it under some padding
+   * which gets cut off as it overflows the parent.
+   * This relies on content box-sizing, although that's the default
+   * we still ensure it does not get overwritten.
    */
-   height: 0.1px;
-   background-color: transparent;
+  padding-bottom: 20px !important;
```

Comment:
Tested in Windows 10 Edge/FF and 20px is definitely enough. 
Most seem to cap-off at 17px http://www.textfixer.com/tutorials/browser-scrollbar-width.php

Documented why 20px in the CSS comments.